docs/migration/seidb_archive_migration.md

```diff
@@ -0,0 +1,177 @@
+###                             SeiDB Configuration                       ###
+#############################################################################
+
+[state-commit]
+# Enable defines if the SeiDB should be enabled to override existing IAVL db backend.
+sc-enable = true
+
+# AsyncCommitBuffer defines the size of asynchronous commit queue, this greatly improve block catching-up
+# performance, <=0 means synchronous commit.
+sc-async-commit-buffer = 100
+
+# SnapshotKeepRecent defines how many memiavl snapshots (beyond the latest one) to keep
+# Recommend to set to 1 to make sure IBC relayers work.
+sc-keep-recent = 1
+
+# SnapshotInterval defines the number of blocks interval the memiavl snapshot is taken, default to 10000 blocks.
+# Adjust based on your needs:
+# Setting this too low could lead to lot of extra heavy disk IO
+# Setting this too high could lead to slow restart
+sc-snapshot-interval = 10000
+
+# SnapshotWriterLimit defines the max concurrency for taking memiavl snapshot
+sc-snapshot-writer-limit = 1
+
+# CacheSize defines the size of the LRU cache for each store on top of the tree, default to 100000.
+sc-cache-size = 100000
+
+[state-store]
+# Enable defines if the state-store should be enabled for historical queries.
+# In order to use state-store, you need to make sure to enable state-commit at the same time.
+# Validator nodes should turn this off.
+# State sync node or full nodes should turn this on.
+ss-enable = true
+
+# DBBackend defines the backend database used for state-store.
+# Supported backends: pebbledb, rocksdb
+# defaults to pebbledb (recommended)
+ss-backend = "pebbledb"
+
+# AsyncWriteBuffer defines the async queue length for commits to be applied to State Store
+# Set <= 0 for synchronous writes, which means commits also need to wait for data to be persisted in State Store.
+# defaults to 100
+ss-async-write-buffer = 100
+
+# KeepRecent defines the number of versions to keep in state store
+# Setting it to 0 means keep everything, default to 100000
+ss-keep-recent = 0
+
+# PruneIntervalSeconds defines the minimum interval in seconds + some random delay to trigger pruning.
+# It is more efficient to trigger pruning less frequently with large interval.
+# default to 600 seconds
+ss-prune-interval = 600
+
+# ImportNumWorkers defines the concurrency for state sync import
+# defaults to 1
+ss-import-num-workers = 1
+```
+
+
+### Step 2: Stop the node and Run SC Migration
+
+```bash
+systemctl stop seid
+seid tools migrate-iavl --target-db SC --home-dir /root/.sei
+```
+
+This may take a couple hours to run. You will see logs of form
+`Start restoring SC store for height`
+
+
+### Step 3: Note down MIGRATION_HEIGHT
+Note down the latest height as outputted from the sc migration. 
+Save it as an env var $MIGRATION_HEIGHT.
+```bash
+MIGRATION_HEIGHT=<>
+```
+
+
+### Step 4: Restart seid with background SS migration
+```bash
+seid start --migrate-iavl --migrate-height $MIGRATION_HEIGHT --chain-id pacific-1
```

Comment:
Shall we be clear to ask people to modify systemd to change to add --migrate-height? And be sure to always run with this flag to start the seid process in the future before migration is fully down